src/Router.php

```diff
@@ -105,13 +105,27 @@ public function setOptions(array $options)
 
         if (!empty($options['routes'])) {
             foreach ($options['routes'] as $path => $routeConfig) {
-                $defaults = isset($routeConfig['defaults']) ? $routeConfig['defaults'] : [];
-                $requirements = isset($routeConfig['requirements']) ? $routeConfig['requirements'] : [];
-                $this->addRoute(new Route($path, $defaults, $requirements));
+                $name = $path;
+                if (array_key_exists('path', $routeConfig)) {
+                    $path = $routeConfig['path'];
+                }
+                $this->addRouteFromConfig($path, $routeConfig, $name);
             }
         }
     }
 
+    /**
+     * @param string $path
+     * @param array $routeConfig
+     */
+    private function addRouteFromConfig($path, array $routeConfig, $name)
+    {
+        $defaults = isset($routeConfig['defaults']) ? $routeConfig['defaults'] : [];
+        $requirements = isset($routeConfig['requirements']) ? $routeConfig['requirements'] : [];
+        $methods = isset($routeConfig['methods']) ? $routeConfig['methods'] : [Request::METHOD_GET];
```

Comment:
Did you find anything regarding the default methods array here?
So that we still accept GET, POST, PUT, DELETE, etc as the default?